glide.yaml

```diff
@@ -72,3 +72,14 @@ import:
   subpackages:
   - compute/metadata
   - internal
+- package: k8s.io/kubernetes
+  version: e4a1ca7105fa9a3075fbf13a3286f8dddd59a658
```

Comment:
Im getting this error which is new to me:
```
[ERROR]	Update failed for google.golang.org/cloud: The Remote does not match the VCS endpoint
```
Looking at https://github.com/GoogleCloudPlatform/gcloud-golang makes me thing the dependency should be 
`cloud.google.com/go` instead in glide.yaml rather than `google.golang.org/cloud`